packages/browser-repl/src/components/shell-output-line.tsx

```diff
@@ -13,9 +13,9 @@ import { ShowCollectionsOutput } from './types/show-collections-output';
 import { CursorOutput } from './types/cursor-output';
 import { CursorIterationResultOutput } from './types/cursor-iteration-result-output';
 import { ObjectOutput } from './types/object-output';
+import { StatsResultOutput } from './types/stats-result-output';
```

Comment:
what should we do in future to avoid forgetting to implement these here?

Reply:
Well … we didn’t forget to implement this, we just didn’t check properly whether it works as intended in the browser repl, I think

I’m not really worried about this being a common occurrence, tbh